### PR TITLE
Update esolver_ks_lcao.cpp

### PR DESCRIPTION
**Pull request by mohanchen**
### Reminder
- [ ] Have you linked an issue with this pull request?
- [ ] Have you added adequate unit tests and/or case tests for your pull request?
- [ ] Have you noticed possible changes of behavior below or in the linked issue?
- [ ] Have you explained the changes of codes in core modules of ESolver, HSolver, ElecState, Hamilt, Operator or Psi? (ignore if not applicable)

### Linked Issue
Fix #...

### Unit Tests and/or Case Tests for my changes
- A unit test is added for each new feature or bug fix.

### What's changed?
- Example: My changes might affect the performance of the application under certain conditions, and I have tested the impact on various scenarios...

### Any changes of core modules? (ignore if not applicable)
- Example: I have added a new virtual function in the esolver base class in order to ...

### CODE REVIEW

**Review by YuLiu98 — APPROVED** [review]

**Review by WHUweiqingzhou — APPROVED** [review]

File: source/module_esolver/esolver_ks_lcao.cpp [hunk 1]
```diff
@@ -1206,10 +1217,11 @@ void ESolver_KS_LCAO<TK, TR>::after_scf(UnitCell& ucell, const int istep)
                                GlobalV::DRANK);
         }
 
+        // where is new? mohan ask 2024-11-28
         delete ekinetic;
     }
 
-    // add by jingan in 2018.11.7
+    //! 19) Wannier 90 function, added by jingan in 2018.11.7
```
Comment (kirk0830) [inline source/module_esolver/esolver_ks_lcao.cpp:1224]:
```suggestion
    //! 19) Wannier 90 function
```

**Review by kirk0830 — CHANGES_REQUESTED** [review]

File: source/module_esolver/esolver_ks_lcao.cpp [hunk 2]
```diff
@@ -1242,8 +1259,10 @@ void ESolver_KS_LCAO<TK, TR>::after_scf(UnitCell& ucell, const int istep)
         std::cout << FmtCore::format(" >> Finish %s.\n * * * * * *\n", "Wave function to Wannier90");
     }
 
-    // add by jingan
-    if (PARAM.inp.calculation == "nscf" && berryphase::berry_phase_flag && ModuleSymmetry::Symmetry::symm_flag != 1)
+    //! 20) berry phase calculations, added by jingan
```
Comment (kirk0830) [inline source/module_esolver/esolver_ks_lcao.cpp:1262]: Note that this PR deletes some developers' names, the following code change can make this PR consistent with itself
```suggestion
    //! 20) berry phase calculations
```

Comment (kirk0830) [inline source/module_esolver/esolver_ks_lcao.cpp:1262]: Note that this PR deletes some developers' names, the following code change can make this PR consistent with itself
```suggestion
    //! 20) berry phase calculations
```

File: source/module_esolver/esolver_ks_lcao.cpp [hunk 3]
```diff
@@ -1054,14 +1054,22 @@ void ESolver_KS_LCAO<TK, TR>::after_scf(UnitCell& ucell, const int istep)
     ModuleBase::timer::tick("ESolver_KS_LCAO", "out_deepks_labels");
 #endif
 
+    //! 10) Perform RDMFT calculations
     /******** test RDMFT *********/
     if ( PARAM.inp.rdmft == true ) // rdmft, added by jghan, 2024-10-17
```
Comment (kirk0830) [inline source/module_esolver/esolver_ks_lcao.cpp:1059]: Note that this PR deletes some developers' names, the following code change can make this PR consistent with itself
```suggestion
    if ( PARAM.inp.rdmft == true ) // rdmft
```

**Review by kirk0830 — CHANGES_REQUESTED** [review]

**Pull request merged**